utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
```

Comment:
I should link http://www.syslinux.org/wiki/index.php?title=ISOLINUX

- Copy "[bios/]core/isolinux.bin" from the Syslinux archive
- For version 5.00 or newer, also copy "[bios/]com32/elflink/ldlinux/ldlinux.c32" from the Syslinux archive
- Create "[isolinux.cfg](http://www.syslinux.org/wiki/index.php?title=Config)"

The script's error messages will report accurately which of those two is still missing, if required.